cgmanifest.json

```diff
@@ -14348,16 +14348,6 @@
         }
       }
     },
-    {
-      "component": {
-        "type": "other",
-        "other": {
-          "name": "ndctl",
-          "version": "65",
-          "downloadUrl": "https://github.com/pmem/ndctl/archive/v65.tar.gz"
```

Comment:
Note, the ndctl version 65 is removed here and not replaced. ndctl version 71.1 is already present in the cgmanifest file below (since that version is present in SPECS-EXTENDED already)